hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java

```diff
@@ -137,6 +137,10 @@
     CONSUMER_BATCH_SOCKET_TIMEOUT("consumer.batch.socket.timeout", 500),
     CONSUMER_FILTERING_ENABLED("consumer.filtering.enabled", true),
 
+    CONSUMER_SUPERVISOR_TYPE("consumer.supervisor.type", "background"),
+    CONSUMER_BACKGROUND_SUPERVISOR_INTERVAL("consumer.supervisor.background.interval", 10_000),
+    CONSUMER_BACKGROUND_SUPERVISOR_UNHEALTHY_AFTER("consumer.supervisor.background.unhealtyAfter", 60_000),
```

Comment:
I like the camel case idea, but maybe this should stick to the lowercase dot-separated configuration style?